### ISSUE TITLE
sin_lt function checking the wrong fake player in the lookup table

### ISSUE THREAD

**Issue by Jeff1Clark** [NOPEname/nn_math#3]
I was playing around with the functions and after putting the player's rotation as the input and executing sin_lt, I realized something odd. When the player's rotation value is negative, instead of the output going negative it returns 0. I first thought maybe it's because the function only supported input between 0 to 360 so I tried adding 180 before setting it as input, making the range 0 to 360. This also didn't work so I checked the datapack and looked into the functions. The "exe" function was all good, it divides the input by 360 and takes the remainder for the actual angle, divides that by 180 and takes it's reminder to later use for the look-up, later checks if the input was bigger than 180 to convert the output to negative, it all made sense. But then I looked at the look-up table. Instead of using the "tmp", the look-up table uses the input itself, causing anything bigger than 180 to output 0. I changed all the functions under look-up, replacing "in" with "tmp" and now it works.

In short; look-up table checks the wrong player for the angle, causing it to malfunction.

**Comment by NOPEname** [comment]
Is this the fix you are talking about? 200440eae5974af03b028b88688d1252c554f005

**Comment by Jeff1Clark** [comment]
Yes, that was it.
